### PR TITLE
Create PM3EasySpecificInstallationInstructions.md

### PR DESCRIPTION
**Pull request by maxieds**
As noted to @doegox in issue #1616 before @iceman1001 closed the issue. Hope this helps some others out now that my PM3 Easy device is running! 😺

### CODE REVIEW

**Comment by iceman1001** [comment]
None of this information is new and not already documented in the current guides.
the correct makefile,   the button trick,  the instructions the flasher gives you.

**Comment by maxieds** [comment]
@iceman1001 
I still contend that it is worthwhile to have a separate, centralized, easy to find resource for PM3 Easy users. The documentation provided by the manufacturer of the Easy devices and their online vendor really, really sucks! There is also some new information that is useful to Easy users that someone pointed out in #1616: the idea of holding down the button throughout the entire flashing procedure.

**Comment by doegox** [comment]
> The documentation provided by the manufacturer of the Easy devices and their online vendor really, really sucks! 

We already did a favor to Easy owners to be able to use this fork made initially exclusively for RDV4 devices, but we're not there to help further those crappy pm3 clones vendors.

> There is also some new information that is useful to Easy users that someone pointed out in #1616: the idea of holding down the button throughout the entire flashing procedure.

New? Really?

image

**Comment by maxieds** [comment]
@doegox 
Ok. You can do what you want. This seems to me like another attempt to get people to buy the RDV4 rather than the more affordable Easy devices by making it **harder**, but of course not impossible, to use the firmware and command line utilities that are actually under active development. It would have helped me very much to have all of that information in a centralized location starting a couple of months ago...

**Comment by mwalker33** [comment]
I think this one can be a little tricky.  The documentation is already on (this) github.  If you follow the "trouble shooting" links it talks about the "button trick" as well as other things to check and review.
I believe the issue is more one of it not being front and center when an end user has an issue.  But everything cant be front and center; as with all things, the need to read to find the answer.

I have been following a lot of these discord and still trying to work out why people see to have issues with the firmware updates.  I suspect most of the time they are non-rdv4 proxmark's and not compiled with the correct options; leading to the main application failing.    At the some time, we have seen bad USB cables and broken pm3 easy's.   I believe this is what @doegox  was trying to refer to.  i.e. the repo cant have defaults that work for both out of the box, as the RDV4 is a different device with more things needed.  This repo was designed to build and support the RDV4 new features; as such are the defaults.  But every effort has been made to support other proxmarks (including the easy)

Given that the proxmark is also meant to be a device for advanced users and can have a steep learning curve for beginners; the documentation layout will be structured to support that user base.

Bottom line the button trick and how to get the pm3 easy running is already in the repo.
